ledger-core/v2/insolar/contract/virtual.go

```diff
@@ -24,6 +24,10 @@ type MethodIsolation struct {
 	State        StateFlag
 }
 
+func (i MethodIsolation) IsZero() bool {
+	return i.Interference.IsZero() && i.State.IsZero()
```

Comment:
Usually, check for one zeroable field would be enough

Reply:
somebody by mistake can call WithState, but forget about WithInterference.

So it's better to check both

Reply:
dont mix
IsValid()
and
IsZero()

your implementation is for IsValid() ;)